.github/CHANGELOG.md

```diff
@@ -4,7 +4,10 @@
 
 ### Improvements
 
-- Move changelog to `.github` and add a changelog reminder.
+* Move changelog to `.github` and add a changelog reminder.
+
+* Add support for Controlled Phase Gate (CPhaseShift).
+  [(#112)](https://github.com/PennyLaneAI/pennylane-lightning/issues/112)
```

Comment:
```suggestion
* Add support for Controlled Phase Gate (`CPhaseShift`).
  [(#114)](https://github.com/PennyLaneAI/pennylane-lightning/pull/114)
```

Reply:
Thanks for updating the log file! I update it only because I was asked to do before creating the pull-request.